Kickstarter-iOS/Views/Cells/SettingsTableViewCell.swift

```diff
@@ -11,16 +11,18 @@ final class SettingsTableViewCell: UITableViewCell, ValueCell, NibLoading {
     super.init(coder: aDecoder)
   }
 
-  func configureWith(value cellType: SettingsCellTypeProtocol) {
+  func configureWith(value cellValue: SettingsCellValue) {
```

Comment:
Do we want to change these to the `configure(with:)` pattern when we touch them?

Reply:
I think this is part of the `ValueCell` protocol, we would have to change it everywhere we use `ValueCell`s. Not sure if this PR is the best place for that?